acquia/acquia_cms
# ACMS-1034: Disable Internal Page Cache in favour of Varnish

**Motivation**
Fixes #[1034](https://backlog.acquia.com/browse/ACMS-1034)

**Proposed changes**
 - Remove page_cache from required module list
 - Disable page_cache module

**Alternatives considered**
NA

**Testing steps**
New Site Install:
- Install site with acquia_cms
- Login into application as administrator
- Head towards - /admin/modules to check Internal Page Cache module is disabled

Existing Site:
- Execute database update
- Login into application as administrator
- Head towards - /admin/modules to check Internal Page Cache module is disabled

**Merge requirements**
